A50-xds-outlier-detection.md

```diff
@@ -159,16 +163,20 @@ The subchannel wrapper will track the latest state update from the underlying su
 
 ### xDS Integration
 
-If the `Cluster` message for an underlying cluster has the `outlier_detection` field set, the `xds_cluster_resolver` will generate an `outlier_detection` LB policy config from it for each priority as the top-level LB policy for that priority. In that case, the child policy config will be the config that would otherwise be at the top level for the priority. When `outlier_detection` is configured, the LB policy tree will look like this when using round robin, and the `outlier_detection` LB policy will be inserted similarly when using ring hash:
+If the `Cluster` message for an underlying cluster has the `outlier_detection` field set, the `xds_cluster_resolver` will generate an `outlier_detection` LB policy config from it for each priority as the top-level LB policy for that priority. Each field in that message will be assigned to the corresponding field in the LB policy config. All other fields will be ignored, and all unsupported ejection algorithms will be considered disabled. In that case, the child policy config will be the config that would otherwise be at the top level for the priority. When outlier detection is configured, the `outlier_detection `policy will be inserted above the `xds_cluster_impl` policy. For example, when using round robin, the LB policy tree will look like this:
 
 ![gRPC Client Architecture Diagram](A50_graphics/grpc_xds_client_architecture.png)
 
 [Link to SVG file](A50_graphics/grpc_xds_client_architecture.svg)
 
+
+#### Validation
+
+The `google.protobuf.Duration` fields `interval`, `base_ejection_time`, and `max_ejection_time` must obey the restrictions in the [`google.protobuf.Duration` documentation](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) and they must have non-negative values. The fields `max_ejection_percent`, `enforcing_success_rate`, `failure_percentage_threshold`, and `enforcing_failure_percentage` must have values less than or equal to `100`. 
```

Comment:
Please be explicit that if any of these "must"s are violated, we will NACK the xDS resource.

Reply:
Done.